src/audio/Kconfig

```diff
@@ -500,6 +500,13 @@ config PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32
 	  Support conversion between 16 bit valid sample size in 32 bit container
 	  and 24 bit valid sample size in 32 bit container
 
+config PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32
```

Comment:
I'm wondering if there would be an harm in adding these 1-to-1 pseudo converters unconditionally? They only add those two entries into the `pcm_func_vc_map[]` array, right?

Reply:
yes, they will be only added to pcm_func_vc_map[]